ci-operator/step-registry/sandboxed-containers-operator/e2e/azure/sandboxed-containers-operator-e2e-azure-workflow.yaml

```diff
@@ -1,10 +1,13 @@
 workflow:
   as: sandboxed-containers-operator-e2e-azure
   steps:
+    env:
+      SLEEP_DURATION: 3h
     pre:
     - chain: sandboxed-containers-operator-ipi-azure-pre
     - chain: sandboxed-containers-operator-pre
     post:
+    - ref: cucushift-installer-wait
```

Comment:
How about adding a comment here with `# TODO: remove this step once this process is automated to avoid unnecessary delay`? (provided it's what we intend to do)